spec/import_spec.rb

```diff
@@ -44,6 +44,41 @@ def upload_file!(name, ext='csv')
     find_button('Import').click
   end
 
+  context "posts index" do
+    before do
+        Author.create!(name: 'John', last_name: 'Doe')
+        Author.create!(name: 'Jane', last_name: 'Roe')
+        add_post_resource({
+          validate: true,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.